### PR TITLE
Conversion to Swift 4.2

### PR DESCRIPTION
**Pull request by jackthmp**
Converts codebase to Swift 4.2

### CODE REVIEW

**Review by mindylou — DISMISSED** [review]
LGTM

File: Recast/Info.plist [hunk 1]
```diff
@@ -20,6 +20,20 @@
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>Fabric</key>
+	<dict>
+		<key>APIKey</key>
+		<string>50d77fbd594f6229f8bb9a3669ce18dce7dac38e</string>
```
Comment (doctor3w) [inline Recast/Info.plist]: Is this supposed to be private?

**Review by doctor3w — COMMENTED** [review]
LGTM

**Comment by doctor3w** [comment]
Completes Issue #441

**Review by mindylou — DISMISSED** [review]

**Review by mindylou — APPROVED** [review]

**Review by doctor3w — APPROVED** [review]

**Pull request merged**